designer/server/src/test/scala/pl/touk/nussknacker/ui/api/OpenAPIExamplesValidator.scala

```diff
@@ -0,0 +1,109 @@
+package pl.touk.nussknacker.ui.api
```

Comment:
It's only used in the test, so maybe we can move it to test utils (to `pl.touk.nussknacker.test.utils` package, like we did with e.g `DesignerTestConfigValidator`)?